Resources/Prototypes/_Goobstation/Changeling/Actions/changeling.yml

```
@@ -0,0 +1,701 @@
+# abilities
+
+# starting
+- type: entity
+  id: ActionEvolutionMenu
+  name: Open evolution menu
+  description: Opens the evolution menu.
+  categories: [ HideSpawnMenu ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)